src/server/qgsserverfilter.h

```diff
@@ -77,9 +77,31 @@ class SERVER_EXPORT QgsServerFilter
      * getFeature requests, sendResponse() might have been called several times
      * before the response is complete: in this particular case, sendResponse()
      * is called once for each feature before hitting responseComplete()
+     *
+     * Note that this function is called recursively call if response is flushed
```

Comment:
This comment has been removed, because it is false: calling `QgsRequestHandler::sendResponse()` call the initial `QgsServerResonse::flush()` and not the decorated one, so there is no recursion.